shiny/api-examples/MarkdownStream/app-core.py

```diff
@@ -0,0 +1,30 @@
+import time
+
+import requests
+
+from shiny import App, ui
+
+app_ui = ui.page_fluid(
+    ui.markdown_stream_ui("shiny-readme"),
```

Comment:
Would `ui.output_markdown_stream` be more consistent with our past naming, perhaps? (I think of Chat as a special case because it's equal parts output and input)